### ISSUE TITLE
Morph Mod Crashes During morph

### ISSUE THREAD

**Issue by Mariojacob7red** [iChun/Morph#1072]
Hi iChun. I'm Jacob. I just gotta say, I love using your mods, especially the morph mod. :D

So, I tried looking around on here to find a topic that has this same issue, but I haven't found one yet. I just downloaded the updated Morph mod the other day and I have the api jar file downloaded too. I'm running the mod on Minecraft 1.12.2 Forge version 14.23.2.2611 (just in case the forge version makes a difference) and what happens for me is that when I load up Minecraft and open a world, everything seems fine like it was before. The only difference is that the first time when I loaded it up, it crashed as I was about to choose a mob to morph into and the second time it crashed after I chose a Minecraft bat to morph into.

All together, I usually only run 35 other mods on my Minecraft instance and I can list them for you if you need to know which ones might possibly be causing the mod to crash Minecraft.

Here's the first...
[crash-2020-05-19_20.51.36-client.txt](https://github.com/iChun/Morph/files/4654247/crash-2020-05-19_20.51.36-client.txt)

...and second crash log if you need it.
[crash-2020-05-19_20.56.15-server.txt](https://github.com/iChun/Morph/files/4654248/crash-2020-05-19_20.56.15-server.txt)

**Comment by iChun** [comment]
Hiya,

You needn't use the API jar. It's used for other developers. The main
download is all you need.

**Comment by Mariojacob7red** [comment]
Ok. Even if I get rid of the api jar file, do you think the mod will still be crashing like it did the other two times?

**Comment by Mariojacob7red** [comment]
Never mind. I just tested it out right now and it worked.

Btw, I hope you don't mind if I make a mod review video on it. I've been planing to making one on this since I heard it got updated the other day.

**Comment by Mariojacob7red** [comment]
One more thing iChun. I just got done looking around the mod and I don't know if this is just me, but when I went into the ability settings of the mod and set the "Hostile Ability Mode" to 1, the mobs kept on attacking me. I did close Minecraft too and reopened it just to make sure that setting I changed got applied to it. Any thoughts on what might be keeping the game from making me think I'm a mob too?

**Comment by iChun** [comment]
That ability probably didn't get ported over properly. I didn't make the
abilities port for 1.12, so I've not had a look at it.

**Comment by Mariojacob7red** [comment]
I had a feeling it was probably something like that. I saw that you said on your mod page that someone else ported the abilities over (and I know you're probably busy with other daily stuff which is more important than a mod).

If I ever teach myself some more coding languages besides java, I could probably help out fixing the mod for you. I don't have any experience making mods, but I want to learn eventually since I love using mods in Minecraft! :D I will say, I have figured out how to modify a mod's internal files and change some data around a tiny bit.
